fastai/vision/data.py

```diff
@@ -77,12 +77,16 @@ def show_batch(x:TensorImage, y:TensorImage, samples, ctxs=None, max_n=10, nrows
     return ctxs
 
 # Cell
-def ImageBlock(cls=PILImage):
+def ImageBlock(
+    cls=PILImage # A class that inherits from `PILBase`
+):
```

Comment:
```suggestion
def ImageBlock(cls:PILBase=PILImage):
```
Can just add that as a typehint instead.